index.js

```diff
@@ -84,24 +91,46 @@ const cssStyles = `
 
 addStylesToHead(cssStyles);
 
+function createMessageListener() {
```

Comment:
Prefix this function to avoid collision with the host codebase, 
example timelyCreateMessageListener
, same with handleCloseButtonClick